File: joseffe-barroso/Python/aula04/for/exercise44.py
quantity = 10
sumNumbers = 0
higherNumber = 0

for index in range (0, quantity, 1):
  number = int(input("Digite um número: "))

  if (index == 0):
    higherNumber = number
  elif (number > higherNumber):
    higherNumber = number

  sumNumbers = sumNumbers + number

avarage = sumNumbers / quantity

print("média", avarage)
print("soma", sumNumbers)
print("maior", higherNumber)

File: joseffe-barroso/Java/src/aula04/For/Exercise44.java
package For;

import java.util.Scanner;

public class Exercise44 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		double i, soma, n, maior, media;

		soma = 0;
		maior = 0;

		for (i = 1; (i <= 10); i++) {
			System.out.println("Digite o " + (int) i + "º número positivo: ");
			n = scanner.nextDouble();

			while (n < 0) {
				System.out.println("O número deve ser positivo!");
				System.out.println("Digite o " + (int) i + " número positivo: ");
				n = scanner.nextDouble();
			}
			if (i == 1) {
				maior = n;
			} else if (n > maior) {
				maior = n;
			}

			soma = soma + n;
			scanner.close();
		}
		media = soma / (i - 1);

		System.out.println("O maior número é: " + maior);
		System.out.println("A soma dos valores é: " + soma);
		System.out.println("A média dos valores é: " + media);
	}

}
